site/en/reference/cli/concepts/tcp.md

```diff
@@ -0,0 +1,147 @@
+# Test TCP Functionality With OT CLI
+
+Thread networks offer several `cli tcp` commands for testing peer-to-peer communication
+using TCP endpoints. The `cli tcp` provides one sample TCP endpoint and one sample
+TCP listener with which all `tcp`commands interact.
+
+Command overviews and the example that follows provide information on initializing
+the example TCP endpoint and TCP listener, how to connect to a peer TCP endpoint,
+and how to send a message to the peer.
+
+## TCP commands
+
+For a list of `tcp` commands, type `help`:
+
+> tcp help
+abort
+benchmark
+bind
+connect
+deinit
+init
+listen
+send
+sendend
+stoplistening
+Done
+
+For descriptions and syntax of all commands, refer to the CLI Command Reference.
+The TCP commands begin alphabetically with
+[tcp abort](https://openthread.io/reference/cli/commands#tcp_abort).
+
+### `init` command
+
+Use the `tcp init` command to initialize the TCP module to begin TCP communication.
+The TCP module can then perform many functions, such as listening for incoming
+connections using the example TCP listener provided by the `tcp` CLI.
+If you want to deinitialize the example TCP listener and the example TCP endpoint,
+you can issue the `tcp deinit` command.
+
+### `bind` command
+
+After you initialize the example TCP endpoint, you can run a `tcp bind`
```

Comment:
Can you rewrite these to not use "you can".

Like this one, dropping "you can" would be fine.